app/models/category.rb

```diff
@@ -33,7 +35,11 @@ class Category < ActiveRecord::Base
   scope :ordered, -> { order('rank ASC') }
   scope :ranked, -> { order('rank ASC') }
   scope :featured, -> { where(front_page: true) }
+  scope :non_featured, -> { where(front_page: true) }
   scope :viewable, -> { where.not(name: 'Common Replies')}
+  scope :is_public, -> { where(public: true) }
+  scope :is_internal, -> { where(public: false) }
```

Comment:
Feel like maybe these scopes should match with the later methods, as well as the `viewable` scope?
`publicly_viewable` & `internally_viewable`